pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -596,7 +598,12 @@ func (c *completedConfig) bootstrapSCC(context genericapiserver.PostStartHookCon
 	}
 
 	for _, scc := range bootstrappolicy.GetBootstrapSecurityContextConstraints(bootstrapSCCGroups, bootstrapSCCUsers) {
-		_, err := securityClient.Security().SecurityContextConstraints().Create(scc)
+		external := &securityv1.SecurityContextConstraints{}
+		if err := securityapiv1.Convert_security_SecurityContextConstraints_To_v1_SecurityContextConstraints(scc, external, nil); err != nil {
```

Comment:
I'll need to think about this, `GetBootstrapSecurityContextConstraints()` should return the v1 SCCs right away, although there's a weird hack at the end which was done for upgrades in case some fields were being added in between OpenShift versions...

Reply:
This approach required the least amt of code change, I was getting into a bowl of spaghetti otherwise (not to say this is right, but wanted to see quickly if it worked).

Reply:
It's not necessarily a bowl of spaghetti, it's just that many tests related to this will start failing, but the change should be trivial. We are creating `v1` SCCs anyway now, please change the relevant methods to return v1 objects as well, I guess there's no reason to do the conversion here.

Reply:
ya, you're right. ok working on it